interfaces/manifest.idl

```diff
@@ -0,0 +1,68 @@
+enum AppBannerPromptOutcome {
+    "accepted",
+    "dismissed"
+};
+[Constructor,
+ Exposed=Window]
+interface BeforeInstallPromptEvent : Event {
+    Promise<PromptResponseObject> prompt();
```

Comment:
PromptResponseObject definition is missing.

Reply:
Yeah, it's also missing in [IDL Index](https://w3c.github.io/manifest/#idl-index) section, I've filed an issue at https://github.com/w3c/manifest/issues/637